webapp/components/wpt-results.html

```diff
@@ -188,7 +188,13 @@
           displayedNodes: {
             type: Array,
             value: []
-          }
+          },
+          // Show a diff column (if there are 2 testRuns).
+          diff: Boolean,
+          filter: {
```

Comment:
I tried to use the filter param as https://diff-param-dot-wptdashboard.appspot.com/results/?label=experimental&diff&filter=R but couldn't make it do anything. Is the intention that this PR should be enough to wire it up?

Reply:
filter is a param that's handled by `/api/diff` but not yet forwarded in the UI. Maybe file an issue if you think that's a feature you want!